src/txn/txn.c

```diff
@@ -717,7 +728,7 @@ __wt_txn_release(WT_SESSION_IMPL *session)
 }
 
 /*
- * __txn_prepare_rollback_restore_hs_update --
+ o __txn_prepare_rollback_restore_hs_update --
```

Comment:
typo

Reply:
Good catch, I have no idea how that got there.